### ISSUE TITLE
Provider Request: hollymoviehd.cc

### ISSUE THREAD

**Issue by An0n-01** [cinepro-org/backend#86]
### Prerequisites

- [x] I have checked the existing issues and this is not a duplicate
- [x] I have read the [README.md](https://github.com/cinepro-org/backend/blob/main/README.md) and ensured that this provider is not already supported
- [x] I have included all relevant details in this request
- [ ] I did not answer truthfully to **ALL** the above checkboxes

### Provider Name

hollymoviehd

### Provider URL

https://hollymoviehd.cc/

### Provider Description

has tons of movies and hosts them.

### Implementation Details

GET: https://hollymoviehd.cc/{title-with-dash-if-space}-{releaseyear}/ to text()
Search for: data-wpnonce="{nonce}" OR data-nonce="{nonce}"--> SHOULD BE SAME (is a number)
Search for: data-streamkey="{key}" OR https://flashstream.cc/download/{key} --> SHOULD BE SAME (is a number)

fetch("https://hollymoviehd.cc/wp-admin/admin-ajax.php", {
  "referrer": "https://hollymoviehd.cc/{title-with-dash-if-space}-{releaseyear}/",
  "body": "action=ajax_getlinkstream&streamkey={key}&nonce={nonce}&imdbid={imdb}&tmdbid={tmdb not required tho}",
  "method": "POST",
});

returns: 
{
    "servers_iframe": {
        "streamsvr": "https://flashstream.cc/embed/mxHnrj7_Ig?e=dUZNUHR0bHk5WWtJZnMyMklCcHRtZkh4TCtpV2ErU09iVWduZ1d5eEd1QT0A"
    },
    "subtitles": [],
    "mq": [
        {
            "keyid": "130ae6O8IUA958b",
            "title": "The.Dark.Knight.2008.720p.BluRay.x264-ESub.Hollymoviehd.mp4",
            "subtitles": [],
            "tag": "",
            "servers_iframe": {
                "streamsvr": "https://flashstream.cc/embed/mxHnrj7_Ig",
                "hydrax": "https://playhydrax.com/?v=JLFoxzuX4"
            }
        }
    ]
}

GET: https://flashstream.cc/embed/mxHnrj7_Ig?e=dUZNUHR0bHk5WWtJZnMyMklCcHRtZkh4TCtpV2ErU09iVWduZ1d5eEd1QT0A to text()
search for:         <input type="hiden" id="csrf_token" value="{csrf}">

fetch("https://flashstream.cc/embed/mxHnrj7_Ig?e={csrf}", {
  "referrer": "https://flashstream.cc/embed/mxHnrj7_Ig?e={csrf}",
  "body": "------WebKitFormBoundarydCA3IPAvFXsmQfTA\r\nContent-Disposition: form-data; name=\"token\"\r\n\r\n\r\n------WebKitFormBoundarydCA3IPAvFXsmQfTA\r\nContent-Disposition: form-data; name=\"csrf_token\"\r\n\r\n{csrf}\r\n------WebKitFormBoundarydCA3IPAvFXsmQfTA--\r\n",
});

returns: {
    "success": true,
    "sources": [
        {
            "file": "//khdiamondcdn.asia/gs/130ae6O8IUA958b", //<-- working mp4
            "label": "MAIN",
            "type": "mp4"
        },
        {
            "file": "/streamsvr/mxHnrj7_Ig/0-27?e=dUZNUHR0bHk5WWtJZnMyMklCcHRtZkh4TCtpV2ErU09iVWduZ1d5eEd1QT0A", // THIS REQUEST NEEDS THE REFFERER: https://flashstream.cc/embed/mxHnrj7_Ig?e={CSRF}
            "label": "LS-25",
            "type": "hls"
        },
        {
            "file": "https://flashstream.cc/pl/130ae6O8IUA958b/0-27?e=dUZNUHR0bHk5WWtJZnMyMklCcHRtZkh4TCtpV2ErU09iVWduZ1d5eEd1QT0A", // THIS REQUEST NEEDS THE REFFERER TOO.
            "label": "GS-25",
            "type": "hls"
        }
    ]
}

### Additional Context

```markdown
note. above the values are for id 155
```
